model/src/main/java/org/cloudfoundry/identity/uaa/zone/IdentityZoneConfiguration.java

```diff
@@ -24,6 +25,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class IdentityZoneConfiguration {
 
+    private LockoutPolicy clientLockoutPolicy = new LockoutPolicy();
```

Comment:
Isn't null the correct backwards compatible value? That way, it can fall back on a global policy.